corelib-search/src/main/java/eu/europeana/corelib/search/service/impl/Main.java

```diff
@@ -0,0 +1,10 @@
+package eu.europeana.corelib.search.service.impl;
+
+public class Main {
```

Comment:
Most likely a local test left over? Don't put this here, create a test instead (even for local testing)